module/VuFind/src/VuFind/Db/Service/CommentsServiceInterface.php

```diff
@@ -95,6 +95,16 @@ public function deleteIfOwnedByUser(int $id, UserEntityInterface|int $userOrId):
      */
     public function deleteByUser(UserEntityInterface|int $userOrId): void;
 
+    /**
+     * Deletes comments by given comment ids and user id
```

Comment:
Add punctuation for consistency.

```suggestion
     * Deletes comments by given comment ids and user id.
```